src/linker/Linker/Driver.cs

```diff
@@ -534,6 +541,7 @@ static void Usage (string msg)
 			Console.WriteLine ("  --strip-resources         Remove XML descriptor resources for linked assemblies. Defaults to true");
 			Console.WriteLine ("  --strip-security          Remove metadata and code related to Code Access Security. Defaults to true");
 			Console.WriteLine ("  --used-attrs-only         Any attribute is removed if the attribute type is not used. Defaults to false");
+			Console.WriteLine ("  --annotate-unseen-callers Mark methods without calls through reflection, assist JIT codegen. Defaults to false");
```

Comment:
I'd suggest this to be more explicit e.g. `--no-reflection-methods` Mark all methods never used using reflection. Defaults to false`